dask/multiprocessing.py

```diff
@@ -236,12 +260,16 @@ def get(
     return result
 
 
-def initialize_worker_process():
+def initialize_worker_process(user_initializer=None, use_default_initializer=True):
     """
     Initialize a worker process before running any tasks in it.
     """
     # If Numpy is already imported, presumably its random state was
     # inherited from the parent => re-seed it.
-    np = sys.modules.get("numpy")
-    if np is not None:
-        np.random.seed()
+    if use_default_initializer:
+        np = sys.modules.get("numpy")
+        if np is not None:
+            np.random.seed()
```

Comment:
Wonder if it would make sense to put this in a `default_initializer` function and then just call it here? Might make it easier for people to compose this in other work more easily and simplify maintenance here

Reply:
Done.